core/src/it/scala/org/ensime/fixture/SearchServiceFixture.scala

```diff
@@ -12,7 +13,7 @@ trait IsolatedSearchServiceFixture extends IsolatedSourceResolverFixture {
     try {
       testCode(config, searchService)
     } finally {
-      searchService.shutdown()
+      Await.ready(searchService.shutdown(), 10.seconds)
```

Comment:
In the tests we can use the akka test utils timefactor. Although I'm not sure what to do about the timeouts in the `main` code, they might be prone to random test failures on slow CI hardware.